language/de-DE/mod_articles.ini

```diff
@@ -48,6 +48,8 @@ MOD_ARTICLES_FIELD_LAYOUT_MULTILINE="Mehrzeilig"
 MOD_ARTICLES_FIELD_LAYOUT_SINGLELINE="Einzeilig"
 MOD_ARTICLES_FIELD_LAYOUT_VERT="Vertikal"
 MOD_ARTICLES_FIELD_LINKTITLES_LABEL="Titel verlinken"
+MOD_ARTICLES_FIELD_MODE_DESC="<em><strong>Normal Modus<br><em><strong></strong></em>zeigt eine statische Liste von Beiträgen an. <br><em><strong>Dynamischer Modus</strong></em>erkennt dynamisch, ob es sich um eine Kategorieansicht handelt, und zeigt die Liste der Beiträge in dieser Kategorie an. Es ist am besten, auf allen Seiten anzuzeigen, da alles dynamisch angezeigt wird."
```

Comment:
MOD_ARTICLES_FIELD_MODE_DESC="<em><strong>Normaler Modus</strong></em> zeigt anhand der Konfiguration eine statische Liste von Beiträgen in den Menüpunkten an, denen das Modul zugewiesen ist.<br><em><strong>Dynamischer Modus</strong></em> das Modul kann normal konfiguriert werden, aber die Option Kategorie wird nicht berücksichtigt. Stattdessen erkennt das Modul dynamisch, ob es sich in einer Kategorieansicht befindet und zeigt die Liste der Artikel innerhalb dieser Kategorie an. Wenn der dynamische Modus gewählt wird, soll das Modul am besten auf allen Seiten zugewiesen werden, da alles dynamisch angezeigt wird."